modules/ROOT/pages/platform/security/single-sign-on.adoc

```diff
@@ -5,49 +5,57 @@
 label:AuraDB-Enterprise[]
 label:AuraDS-Enterprise[]
 
-Aura Enterprise supports Single Sign-On (SSO) at both the Console level and for accessing Workspace, Bloom and Browser clients directly at the instance level.
+== SSO levels
+
+Aura admins can configure organization and tenant level SSO:
```

Comment:
We have two admins, tenant and org admin. I don't think we ever define what an Aura admin is. This should probably just say Organization admins can configure...